cucoslib/workers/recommender.py

```diff
@@ -431,11 +456,134 @@ def find_relative_similarity(self, input_stack, input_stack_vectors, filtered_re
 class RecommendationTask(BaseTask):
     _analysis_name = 'recommendation'
     description = 'Get Recommendation'
+    
+    def call_janus_model(self, user_profile):
+        """Calls the Janus model with the user profile information
+            to get the user_persona"""
+        _janus_end_point = "http://janus-demo-janus.dev.rdu2c.fabric8.io/api/v1/janus"
+        try:
+            if user_profile is not None:
+                response = get_session_retry().post(_janus_end_point, json=user_profile)
+                if response.status_code != 200:
+                    _logger.error ("HTTP error {}. Error retrieving Janus data.".format(response.status_code))
```

Comment:
All tasks have logger ready in `self.log` property. Please use it as we do additional configuration to properly aggregate logs. Here and other places as well.